stylua-vscode/src/util.ts

```diff
@@ -21,8 +22,20 @@ type GithubRelease = {
   html_url: string;
 };
 
-const getLatestRelease = async (): Promise<GithubRelease> => {
-  return await fetch(RELEASES_URL).then((r) => r.json());
+const getRelease = async (version: string): Promise<GithubRelease> => {
+  if (version === "latest") {
+    return await (await fetch(RELEASES_URL + "/latest")).json();
```

Comment:
This is the only place that we modify the URL. We could update the worker to handle this. Although, I would still like to add github authentication in a separate PR.